core/src/com/po/fuck/model/GeometryMisc.java

```diff
@@ -80,16 +80,10 @@ public static Polygon createRectangle(Vector2 position, float width, float heigh
         return polygon;
     }
 
-    public static Vector2 getPointPositionOnFlippedSprite(Vector2 spritePosition, Vector2 spriteDirection, Vector2 point, boolean flipped) {
-        Vector2 pointRealPosition = new Vector2(point);
-        pointRealPosition.rotateDeg(spriteDirection.angleDeg());
+    public static Vector2 getPointPositionOnFlippedSprite(Vector2 spritePosition, Vector2 spriteRotation, Vector2 point, boolean flipped) {
```

Comment:
`Vector2 spriteRotation` - should be float in `float rotationRad`/`float rotationDeg` or smth like this.